crates/sui-core/src/authority_server.rs

```diff
@@ -924,7 +934,12 @@ impl ValidatorService {
         let epoch_store = self.state.load_epoch_store_one_call_per_task();
         let certificate = request.into_inner();
         certificate.validity_check(epoch_store.protocol_config(), epoch_store.epoch())?;
-
+        for client in self.trusted_certificate_forwarder.clone() {
+            let certificate = certificate.clone();
+            spawn_monitored_task!(async move {
+                let _ = client.handle_certificate_v2(certificate, None).await;
```

Comment:
Wrap this in a timeout, and `info` log the errors?

Reply:
done